client/src/components/organisms/RequestGroupForm.tsx

```diff
@@ -12,8 +12,11 @@ import RichTextField from "../atoms/RichTextField";
 import { RootState } from "../../data/reducers";
```

Comment:
Do we need to add the file input to this component? there seems to be a test component `FileUploadTest` for `UploadThumbnailService`